Polyhedron Demo: CMake error with CMake development branch 3.9.20170901

The test platforms have been updated during the week-end, and ArchLinux has a new bleeding-edge version of CMake, that triggers CMake errors with the polyhedron demo.

https://cgal.geometryfactory.com/CGAL/testsuite/results-4.12-Ic-31.shtml#Polyhedron_Demo
```
CMake Error in CMakeLists.txt:
  Cannot find source file:

    File_loader_dialog_moc_moc.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx


CMake Error in CMakeLists.txt:
  Cannot find source file:

    Viewer_moc.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx


CMake Error in Plugins/IO/CMakeLists.txt:
  Cannot find source file:

    /home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Polyhedron_Demo/Plugins/IO/Polyhedron_demo_statistics_on_polyhedron_dialog.ui

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx

```

CMake is right: those files do not exist, but they are referenced in the `CMakeLists.txt` files.

Please create a patch on the CGAL-4.11-branch.

--- a/Polyhedron/demo/Polyhedron/CMakeLists.txt
+++ b/Polyhedron/demo/Polyhedron/CMakeLists.txt
@@ -145,9 +145,7 @@ if(CGAL_Qt5_FOUND AND Qt5_FOUND AND OPENGL_FOUND AND QGLVIEWER_FOUND)
   qt5_wrap_ui( Show_point_dialogUI_FILES Show_point_dialog.ui )
   qt5_wrap_ui( ViewerUI_FILES  ImageInterface.ui)
   qt5_generate_moc( "File_loader_dialog.h" "${CMAKE_CURRENT_BINARY_DIR}/File_loader_dialog_moc.cpp" )
-  add_file_dependencies( File_loader_dialog_moc_moc.cpp "${CMAKE_CURRENT_SOURCE_DIR}/File_loader_dialog.h" )
-
-  add_file_dependencies( Viewer_moc.cpp "${CMAKE_CURRENT_SOURCE_DIR}/Viewer.h" )
+  add_file_dependencies( File_loader_dialog_moc.cpp "${CMAKE_CURRENT_SOURCE_DIR}/File_loader_dialog.h" )
 
   include( ${CMAKE_CURRENT_SOURCE_DIR}/polyhedron_demo_macros.cmake )
 

--- a/Polyhedron/demo/Polyhedron/Plugins/IO/CMakeLists.txt
+++ b/Polyhedron/demo/Polyhedron/Plugins/IO/CMakeLists.txt
@@ -10,7 +10,7 @@ target_link_libraries(io_implicit_function_plugin scene_implicit_function_item)
 polyhedron_demo_plugin(nef_io_plugin Nef_io_plugin)
 target_link_libraries(nef_io_plugin  scene_nef_polyhedron_item)
 
-qt5_wrap_ui( statisticsPolyhedronUI_FILES Polyhedron_demo_statistics_on_polyhedron_dialog.ui)
+qt5_wrap_ui( statisticsPolyhedronUI_FILES Statistics_on_polyhedron_dialog.ui)
 polyhedron_demo_plugin(off_plugin OFF_io_plugin)
 target_link_libraries(off_plugin scene_polyhedron_item scene_polygon_soup_item scene_points_with_normal_item)
 

